[2022-04-13 03:35:43,946] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: first_dag.hello_task 2022-04-13T03:34:43.660902+00:00 [queued]>
[2022-04-13 03:35:43,967] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: first_dag.hello_task 2022-04-13T03:34:43.660902+00:00 [queued]>
[2022-04-13 03:35:43,967] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 03:35:43,968] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2022-04-13 03:35:43,968] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 03:35:43,978] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): hello_task> on 2022-04-13T03:34:43.660902+00:00
[2022-04-13 03:35:43,982] {standard_task_runner.py:52} INFO - Started process 46 to run task
[2022-04-13 03:35:43,986] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'hello_task', '2022-04-13T03:34:43.660902+00:00', '--job-id', '34', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp38kwcjwo', '--error-file', '/tmp/tmp5fsvokk8']
[2022-04-13 03:35:43,987] {standard_task_runner.py:77} INFO - Job 34: Subtask hello_task
[2022-04-13 03:35:44,051] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.hello_task 2022-04-13T03:34:43.660902+00:00 [running]> on host cea7f5219bb8
[2022-04-13 03:35:44,132] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=hello_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-13T03:34:43.660902+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-13T03:34:43.660902+00:00
[2022-04-13 03:35:44,134] {python.py:118} INFO - Done. Returned value was: Hello world from first Airflow DAG!
[2022-04-13 03:35:44,177] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=hello_task, execution_date=20220413T033443, start_date=20220413T033543, end_date=20220413T033544
[2022-04-13 03:35:44,218] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-13 03:35:44,242] {local_task_job.py:146} INFO - Task exited with return code 0
